schedule/yast/maintenance/autoupgrade_sles12sp5_to_sles15spx_maintenance.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: autoupgrade
+description: >
+  Performs a migration from installed system with released maintenance updates to mu unreleased testsuites.
```

Comment:
`mu` is not widely used, `maintenance updates`. It should say `...to a target system using its unreleased maintenance updates'